nixos/lib/test-driver/default.nix

```diff
@@ -25,6 +25,8 @@ python3Packages.buildPythonApplication rec {
   checkPhase = ''
     mypy --disallow-untyped-defs \
           --no-implicit-optional \
+          --pretty \
+          --no-color-output \
```

Comment:
I disabled the color output because for some reason the ansi color codes are broken:
```
Success: no issues found in 6 source files(B
```
With this change, the `(B` is not printed anymore.